### PR TITLE
Added blockchain provider section

### PR DESCRIPTION
**Pull request by shelleyolivia**
Hey there - as this page talks about blockchain providers, I added a section dedicated to this to make it clear to readers. I work at Alchemy so I added Alchemy as I am familiar with its functionality and offerings. I added context to let readers understand the benefits of using a 3rd party API like Alchemy and also give more details to help readers determine if Alchemy's product offerings was a good fit for what they were interested in building. 

Also, I noticed that ropsten is being used in the rpc urls, I took them out where it made sense. 

Let me know of any questions, thanks!

### CODE REVIEW

File: docs/sdk/web/providers/evm.mdx [hunk 1]
```diff
@@ -47,6 +47,11 @@ await web3auth.initModal();
 
 web3auth.connect();
 ```
+## Blockchain Providers
+
+- [Alchemy](https://www.alchemy.com/). As a developer platform, Alchemy provides a suite of developer toolings and abstractions including JSON RPC APIs across all major chains, an ethers.js SDK, and a library of [enhanced APIs](https://docs.alchemy.com/reference/enhanced-apis-overview) like their NFT APIs. 
+
```
Comment (yashovardhan) [inline docs/sdk/web/providers/evm.mdx:53]: we should add other providers like infura, quicknode and ankr here.

Comment (shelleyolivia) [reply docs/sdk/web/providers/evm.mdx:53]: Sg, just changed it :) Didn't want to misspeak on their details

**Comment by yashovardhan** [comment]
Hey @shelleyolivia 

We looked further into how we can add Alchemy and other providers in our documentation, we thought it is the best to add them in the main doc page about connecting blockchain rather than SDK reference. We'll add them there in this PR: https://github.com/Web3Auth/web3auth-docs/pull/214


Thanks a lot for your PR though, we'll add the content in the appropriate place. Closing it for now.

**Comment by shelleyolivia** [comment]
Gotcha @yashovardhan, appreciate the context! Appreciate your guys reviewing, your docs are fire :)

**Pull request closed without merging**